domain/src/main/java/com/thoughtworks/go/domain/materials/git/GitMaterialUpdater.java

```diff
@@ -159,7 +159,7 @@ private BuildCommand cmdClone(String workDir, int cloneDepth) {
         if (material.isShallowClone()) {
             cloneArgs.add(format("--depth=%s", String.valueOf(cloneDepth)));
         }
-        cloneArgs.add(material.getUrlArgument().forCommandline());
+        cloneArgs.add(material.getUrlArgument().rawUrl());
```

Comment:
This should be changed back to forCommandline()

Reply:
done